internal/commands/executor.go

```diff
@@ -24,6 +27,10 @@ type Executor interface {
 	Account() service.Account
 	Plan() service.Plans
 	All() internal.AllServices
+	Debug(msg string, args ...interface{})
+	Info(msg string, args ...interface{})
+	Error(msg string, args ...interface{})
+	WithLogger(args ...interface{}) Executor
```

Comment:
Not sure about these, but went with 'as close to flume as possible'. Error(), for example, doesn't make it obviously clear that it's a error *logging* method. Maybe DebugLog / LogDebug etc? Or maybe executor could just provide Logger() flume.logger method?